net/snort3/files/local.lua

```diff
@@ -50,3 +50,10 @@ file_policy = {
     }
   }
 }
+
```

Comment:
I would suggest to split these changes to separate commit.

Reply:
done!